src/MeshKernelNET/Native/Mesh2DNative.cs

```diff
@@ -70,11 +70,21 @@ public struct Mesh2DNative
         /// </summary>
         public int num_nodes;
 
+        /// <summary>
+        /// The number of valid mesh nodes
+        /// </summary>
+        public int num_valid_nodes;
```

Comment:
the number of valid nodes can be computed, there is no need to make it a member.